src/fsharp/absil/illib.fs

```diff
@@ -1127,11 +1127,9 @@ type LayeredMap<'Key, 'Value when 'Key : comparison> = Map<'Key, 'Value>
 [<AutoOpen>]
 module MapAutoOpens =
     type Map<'Key, 'Value when 'Key : comparison> with
-
+        
         static member Empty : Map<'Key, 'Value> = Map.empty
-
-        member x.Values = [ for KeyValue(_, v) in x -> v ]
```

Comment:
@dsyme Do we expect all FCS clients to use FSharp.Core preview now? I don't think it's a good idea to tie FCS and FSharp.Core previews like this.

The tooling now shows errors (https://github.com/dotnet/fsharp/issues/12142) and it seems we won't be able to run FCS with a stable FSharp.Core version anymore?

Reply:
@auduchinok Hmmmm could you please open a separate issue about this.  I agree public non-pre-release FCS should indeed only use a public non-pre-release FCS.  

However if FCS in pre-release and the release plan is matching FSharp.Core than it is ok